index.html

```diff
@@ -2866,32 +2836,57 @@ <h3><dfn>Writing or pushing content</dfn></h3>
         <a href="https://www.w3.org/2001/tag/doc/URNsAndRegistries-50.html">
         W3C TAG recommendations</a> to avoid using URNs in the Web.
         Therefore, Web NFC does not use the URN prefix neither when reading
-        or writing <a>NDEF messages</a>, but requires the leading colon ('`:`')
-        before the type name.
+        or writing <a>NDEF messages</a>.
       </p>
       <p>
-        To <dfn>validate external type</dfn> given a |input:USVString| run
+        To <dfn>validate external type</dfn> given |input:USVString|, run
         these steps:
       </p>
       <ol class=algorithm id="validate-external-type">
         <li>
           If |input:string| is not a {{USVString}} or it is empty, or its
-          length exceeds 255 bytes, or it contains white spaces,
-          return `false`.
+          length exceeds 255 bytes, return `false`.
         </li>
         <li>
-          If |input| does not start with colon ('`:`'), return |input|.
-          <p class="issue">
-            Validating domain name needs an algorithm.
-          </p>
+          Find the last occurrence of `U+003A` (`:`) in |input|.
+          If not found, return `false`.
+        </li>
+        <li>
+          Let |domain| be the part of |input| preceding the last colon,
```

Comment:
cannot refer to colon here. Need to store the index above.

Reply:
Let colonIndex be the last ...

Reply:
Yeah, it's still pseudocode, and it's not more clear than before (when we just want to say use the last colon), but fine.

Reply:
Fixed.